mu4/scenes/inspector/view/inspector_resources.qrc

```diff
@@ -0,0 +1,157 @@
+<RCC>
+    <qresource prefix="/">
+        <file>qml/MuseScore/Inspector/general/GeneralInspectorView.qml</file>
+        <file>qml/MuseScore/Inspector/InspectorForm.qml</file>
+        <file>qml/MuseScore/Inspector/notation/NotationInspectorView.qml</file>
+        <file>qml/MuseScore/Inspector/notation/notes/StemSettings.qml</file>
+        <file>qml/MuseScore/Inspector/notation/notes/NoteSettingsTabPanel.qml</file>
+        <file>qml/MuseScore/Inspector/notation/notes/HeadSettings.qml</file>
+        <file>qml/MuseScore/Inspector/notation/notes/internal/NoteheadsGrid.qml</file>
+        <file>resources/icons/raster-vertical.svg</file>
```

Comment:
What are the svg icons for in inspector resources?

Reply:
The icons that are not in the icon-font yet (they wait for the big icon-font update)